README.md

```diff
@@ -48,6 +53,13 @@ epicli apply -f demo.yaml
 ```
 You will be asked for a password that will be used for encryption of some of build artifacts. More information [here](docs/home/howto/SECURITY.md#how-to-run-epicli-with-password)
 
+Since version 0.7 epicli has an option to backup/recovery almost all its components. More information [here](https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/howto/BACKUP.md)
```

Comment:
I think it's not accurate at all. We have to be more humble and honest here. The "almost all" sounds here so fabulous, but in reality we support just a fraction of what we should have here. Also things like backup of elasticsearch clusters is not implemented correctly (it's currently blocked). I'd put an actual list here. :+1:

Reply:
I think `some` would be good here due to a list of supported components is already in the provided link.